test/ffmpeg-qsv/encode/vp9.py

```diff
@@ -68,3 +67,61 @@ def test(self, case, gop, bitrate, fps, slices, refmode, quality, looplvl, loops
     self.init(spec, case, gop, bitrate, fps, slices, quality)
     self.encode()
 
+class cqp(VP9_8EncoderTest):
+  def init(self, tspec, case, ipmode, qp, quality, slices):
+    vars(self).update(tspec[case].copy())
+    vars(self).update(
+      case      = case,
+      gop       = 30 if ipmode != 0 else 1,
+      qp        = qp,
+      slices    = slices,
+      rcmode    = "cqp",
+      quality   = quality,
+    )
+
+  @parametrize_with_unused(*gen_vp9_cqp_lp_parameters(spec), ['refmode', 'loopshp', 'looplvl'])
```

Comment:
Oops, I missed this one.  We need a new case generator for "cqp" variants.  Please submit a new PR to fix these.